### PR TITLE
[13.0] [ADD] Account Multi VAT Sale

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
Depends on https://github.com/OCA/account-fiscal-rule/pull/102

### CODE REVIEW

**Review by chrisdec — APPROVED** [review]
Functional tests

**Review by sanrav — APPROVED** [review]
Functional testing OK

**Comment by sbidoul** [comment]
LGTM technically except a few details, but I'd rather see it live in OCA/account-fiscal-rule: it will be easier to discover there and it's also closely linked to the features in account-fiscal-rule.

File: .isort.cfg [hunk 1]
```diff
@@ -10,3 +10,4 @@ known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
 default_section=THIRDPARTY
+known_third_party=setuptools
```
Comment (sbidoul) [inline .isort.cfg]: It should not be necessary to modify `.isort.cfg` ever.

File: .travis.yml [hunk 2]
```diff
@@ -37,6 +37,10 @@ env:
     - VERSION="13.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
 
 install:
+  #  FIXME:
+  - git clone --branch 13.0-add_account_multi_vat_tbi
+    https://github.com/acsone/account-fiscal-rule.git PR102
+  - rm -rf account_multi_vat && cp -ar PR102/account_multi_vat .
```
Comment (sbidoul) [inline .travis.yml]: Remove before merge.

File: setup/account_multi_vat_sale/.eggs/README.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
+
+This directory caches those eggs to prevent repeated downloads.
+
+However, it is safe to delete this directory.
+
```
Comment (sbidoul) [inline setup/account_multi_vat_sale/.eggs/README.txt]: The .eggs directory should not be in git. I'd recommend you put it in your global gitignore (`git config --global core.excludesFile '~/.gitignore'`).

**Comment by gurneyalex** [comment]
@ThomasBinsfeld  the pr you depend on has been merged. Can you update this one as requested by stéphane?

**Comment by ThomasBinsfeld** [comment]
I moved the module to oca/account-fiscal-rule. @chrisdec @sanrav @sbidoul could you update your review there? https://github.com/OCA/account-fiscal-rule/pull/155

**Pull request closed without merging**